src/components/legal/privacy-policy/privacy-policy.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { View, TouchableOpacity, Platform } from 'react-native';
+import { withTheme, IThemeProps } from '../../../core/theme/with-theme';
+import stylesProvider from './styles';
+import { smartConnect } from '../../../core/utils/smart-connect';
+import Icon from '../../icon';
+import { ICON_SIZE, BASE_DIMENSION } from '../../../styles/dimensions';
+import { LoadingIndicator } from '../../loading-indicator/loading-indicator';
+import { WebView } from '../../../library/webview/webview';
+
+interface IExternalProps {
+    onClose?: () => void;
+    showClose?: boolean;
+}
+
+export const PrivacyPolicyComponent = (
+    props: IExternalProps & IThemeProps<ReturnType<typeof stylesProvider>>
+) => {
+    return (
+        <View
+            style={[
+                props.styles.container,
+                { paddingTop: Platform.OS === 'ios' && props.showClose ? BASE_DIMENSION * 5 : 0 }
```

Comment:
android and web don't have this problem?

Reply:
The LegalModal it's not shown on the web Platform.
On Android it's working, but it has to be tested and verified on a device with notch.

On iOS we have the following issue:

![image](https://user-images.githubusercontent.com/32679726/75977938-1968a980-5ee6-11ea-9430-a14819096490.png)